packages/wonder-blocks-cell/src/components/detail-cell.js

```diff
@@ -0,0 +1,86 @@
+// @flow
+import * as React from "react";
+import {StyleSheet} from "aphrodite";
+
+import Color from "@khanacademy/wonder-blocks-color";
+import {Strut} from "@khanacademy/wonder-blocks-layout";
+import Spacing from "@khanacademy/wonder-blocks-spacing";
+import {LabelSmall, LabelMedium} from "@khanacademy/wonder-blocks-typography";
+
+import CellCore from "./internal/cell-core.js";
+
+import type {CellProps, TypographyText} from "../util/types.js";
+
+type DetailCellProps = {|
+    ...CellProps,
+
+    /**
+     * You can either provide a string or a custom node Typography element (or
+     * nothing at all). Both a string or a custom node Typography element will
+     * occupy the “Subtitle1” area of the Cell.
+     */
+    subtitle1?: TypographyText,
+
+    /**
+     * You can either provide a string or a custom node Typography element (or
+     * nothing at all). Both a string or a custom node Typography element will
+     * occupy the “Subtitle2” area of the Cell.
+     */
+    subtitle2?: TypographyText,
+|};
+
+/**
+ * This is a variant of BasicCell that allows adding subtitles, before and after
+ * the cell title.
+ *
+ * ### Usage
+ *
+ * ```jsx
+ * import {DetailCell} from "@khanacademy/wonder-blocks-cell";
+ *
+ * <DetailCell
+ *  leftAccessory={<Icon icon={icons.contentVideo} size="medium" />}
+ *  subtitle1="Subtitle 1"
+ *  title="Detail cell"
+ *  subtitle1="Subtitle 2"
+ *  rightAccessory={<Icon icon={icons.caretRight} size="medium" />}
+ * />
+ * ```
+ */
+function DetailCell(props: DetailCellProps): React.Node {
+    const {title, subtitle1, subtitle2, ...coreProps} = props;
+
+    const maybeRenderSubtitle = (subtitle?: TypographyText): React.Node => {
+        if (!subtitle) {
+            return null;
+        }
+
+        if (typeof subtitle === "string") {
+            return <LabelSmall style={styles.subtitle}>{subtitle}</LabelSmall>;
+        }
+
+        return subtitle;
+    };
+
+    return (
+        <CellCore {...coreProps}>
+            {maybeRenderSubtitle(subtitle1)}
+            {typeof title === "string" ? (
+                <LabelMedium>{title}</LabelMedium>
+            ) : (
+                title
+            )}
+            {/* Add a vertical spacing between the title and the subtitle */}
+            {subtitle2 && <Strut size={Spacing.xxxxSmall_2} />}
```

Comment:
I think we need to land on final values for these spacing values.  https://khanacademy.atlassian.net/browse/MOB-4368

Reply:
Also, it should probably be the same as the vertical spacing between the above one and the title. (cc @megaroeny )

Reply:
I will double-check that what is in the Figma `.base` component is what we want. But need to confirm with you and Juan. Let's quickly review together tomorrow if you have a brief moment. I think I wanted to have it be 2px above/below the title, especially with letters like a _g_, it's descender/tail was too close to the title.

Reply:
Yeah, that was the main reasoning about using 2px. We tested that directly in the browser and ended up picking `2px` on the subtitle because it rendered better with the cases that @megaroeny mentioned.